airflow/config_templates/default_test.cfg

```diff
@@ -60,6 +60,12 @@ hide_paused_dags_by_default = False
 [email]
 email_backend = airflow.utils.email.send_email_smtp
 
+[hipchat]
+token = 42
```

Comment:
These should be connection_ids, not the direct token as these are effectively passwords and we don't want them "in-the-plain" in the config file.

Or alternatively add support for `token_cmd` to run a command to get these values.